src/app/core/signals/remote_signals/wallet.nim

```diff
@@ -3,6 +3,8 @@ import json, options
 import base
 import signal_type
 
+const SingTransactionsEventType* = "sing-transactions"
```

Comment:
Typo `Sing` -> `Sign`

Reply:
There is a typo on the status-go side as well, but will update that through some other issue, cause that part for this PR is already merged.